### PR TITLE
Add support for forecast-only low resolution

### PR DESCRIPTION
**Pull request by JessicaMeixner-NOAA**
**Description**

This PR ensures that we can run C96 5deg and C48 5 deg for 2021032312. 

ICs were staged on hera from @sanatcumar and @guillaumevernieres  so that this can be run "out of the box", but will have to be transferred to other machines if desired. 

This PR requires two fix file updates: 
https://github.com/NOAA-EMC/global-workflow/issues/1647
https://github.com/NOAA-EMC/global-workflow/issues/1645
that must be completed before this is merged. 

Thank you to: @sanatcumar @barlage @HelinWei-NOAA Bing Fu @jiandewang @guillaumevernieres  @aerorahul @NeilBarton-NOAA  and others who helped provide input, fix files, directions, and other help! 

**Type of change**

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

**How Has This Been Tested?**

- [x] Hera C96 5deg S2S forecast only 2021032312 see:  /scratch1/NCEPDEV/climate/Jessica.Meixner/lowresgw/newtest/c96o500 
 *NOTE* The default for C96 is 1 deg therefore you must change your config.base to have the correct ocean resolution 
- [x] Hera C48 5deg S2S forecast only 2021032312 see:  /scratch1/NCEPDEV/climate/Jessica.Meixner/lowresgw/newtest/c48o500 
- [x] Hera C384 .25deg S2S forecast only 20130101 see:  /scratch1/NCEPDEV/climate/Jessica.Meixner/lowresgw/newtest/c384o025 

  
**Checklist**

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] Any dependent changes have been merged and published

### CODE REVIEW

File: workflow/rocoto/workflow_tasks.py
Comment (WalterKolczynski-NOAA) [inline workflow/rocoto/workflow_tasks.py]: You can just delete the old code.

**Review by WalterKolczynski-NOAA — CHANGES_REQUESTED** [review]
Looks good so far.

File: workflow/rocoto/workflow_tasks.py
Comment (JessicaMeixner-NOAA) [reply workflow/rocoto/workflow_tasks.py]: Done!

**Comment by JessicaMeixner-NOAA** [comment]
There's been an update removing a "to do" item in this PR from @jiandewang who removed the 025 resolution dependency in the ocean post.    The only thing that changes is meta-data.  The three tests with this new output can be found here: 
/scratch1/NCEPDEV/climate/Jessica.Meixner/lowresgw/oceanupdate/c*o* for the various resolutions.

**Comment by WalterKolczynski-NOAA** [comment]
New cpl fix version 20230526 created for this update. Orog fix changes was just additions, so existing version (20220805) is still used.

**Comment by JessicaMeixner-NOAA** [comment]
I have updated the version for cpl and am currently working to test a fresh clone for out of the box runs on hera.  Note the ICs are currently only staged on hera, so if these tests are desired for other machines we need to have @jiandewang move them to WCOSS2 and others to move to orion into the IC staged area.  

Please let me know what else I can do for this PR.  Note, I also just finished adding a low res wave capability to this, which should get a new PR for that soon as well.

**Comment by JessicaMeixner-NOAA** [comment]
Latest test will be here: /scratch1/NCEPDEV/climate/Jessica.Meixner/lowresgw/usefixfromofficial/c48o500

**Review by WalterKolczynski-NOAA — APPROVED** [review]
Approved pending CI testing

**Comment by JessicaMeixner-NOAA** [comment]
> Automated global-workflow Testing Results:
> 
> ```
> Machine: Orion
> Start: Tue May 30 00:40:24 CDT 2023 on Orion-login-1.HPC.MsState.Edu
> ---------------------------------------------------
> Checkout:                      *SUCCESS*
> Checkout: Completed at Tue May 30 00:42:05 CDT 2023
> Build:                         *SUCCESS*
> Build: Completed at Tue May 30 01:52:31 CDT 2023
> Created experiment:            *SUCCESS*
> Case setup: Completed at Tue May 30 01:52:38 CDT 2023 for experiment C96_atm3DVar
> Experiment C96_atm3DVar Terminated: *FAILED*
> Experiment C96_atm3DVar Terminated with 1 tasks failed at Tue May 30 03:21:07 CDT 2023
> Error logs:
> /work2/noaa/stmp/GFS_CI_ROOT/PR/1648/RUNTESTS/C96_atm3DVar/COMROT/C96_atm3DVar/logs/2021122100/gdasvrfy.log
> ```

The error is: 
```
+ radmon_verf_angle.sh[215](2021122100): gzip radmon_angle.tar
+ radmon_verf_angle.sh[216](2021122100): mv radmon_angle.tar.gz /work/noaa/global/mterry/monitor/radmon/stats/C96_atm3DVar/gdas.20211221/00/.
mv: cannot move ‘radmon_angle.tar.gz’ to a subdirectory of itself, ‘/work/noaa/global/mterry/monitor/radmon/stats/C96_atm3DVar/gdas.20211221/00/./radmon_angle.tar.gz’
+ radmon_verf_angle.sh[1](2021122100): postamble radmon_verf_angle.sh 1685434259 1

```

However I'm not sure how any of the changes I made changed something here?   I'll sync in the develop to this branch in case that's the issue?

**Comment by WalterKolczynski-NOAA** [comment]
> > Automated global-workflow Testing Results:
> > ```
> > Machine: Orion
> > Start: Tue May 30 00:40:24 CDT 2023 on Orion-login-1.HPC.MsState.Edu
> > ---------------------------------------------------
> > Checkout:                      *SUCCESS*
> > Checkout: Completed at Tue May 30 00:42:05 CDT 2023
> > Build:                         *SUCCESS*
> > Build: Completed at Tue May 30 01:52:31 CDT 2023
> > Created experiment:            *SUCCESS*
> > Case setup: Completed at Tue May 30 01:52:38 CDT 2023 for experiment C96_atm3DVar
> > Experiment C96_atm3DVar Terminated: *FAILED*
> > Experiment C96_atm3DVar Terminated with 1 tasks failed at Tue May 30 03:21:07 CDT 2023
> > Error logs:
> > /work2/noaa/stmp/GFS_CI_ROOT/PR/1648/RUNTESTS/C96_atm3DVar/COMROT/C96_atm3DVar/logs/2021122100/gdasvrfy.log
> > ```
> 
> The error is:
> 
> ```
> + radmon_verf_angle.sh[215](2021122100): gzip radmon_angle.tar
> + radmon_verf_angle.sh[216](2021122100): mv radmon_angle.tar.gz /work/noaa/global/mterry/monitor/radmon/stats/C96_atm3DVar/gdas.20211221/00/.
> mv: cannot move ‘radmon_angle.tar.gz’ to a subdirectory of itself, ‘/work/noaa/global/mterry/monitor/radmon/stats/C96_atm3DVar/gdas.20211221/00/./radmon_angle.tar.gz’
> + radmon_verf_angle.sh[1](2021122100): postamble radmon_verf_angle.sh 1685434259 1
> ```
> 
> However I'm not sure how any of the changes I made changed something here? I'll sync in the develop to this branch in case that's the issue?

@EdwardSafford-NOAA would you please take a look at this failure? It seems unrelated to @JessicaMeixner-NOAA's PR, but CI tests passed before this. Might just be a fluke this PR hit it, but it needs to be fixed.

**Comment by WalterKolczynski-NOAA** [comment]
Oops, hit the wrong button.

@EdwardSafford-NOAA @JessicaMeixner-NOAA I'm going to rerun the CI test, so I copied the rotdir and radmon directories to `/work2/noaa/stmp/wkolczyn/PR1648` to continue examination.

**Comment by EdwardSafford-NOAA** [comment]
@WalterKolczynski-NOAA I agree this error is likely unrelated to changes in this PR.

**Comment by JessicaMeixner-NOAA** [comment]
@WalterKolczynski-NOAA @EdwardSafford-NOAA the CI seems to be working, but I'm not sure what changed other than merging in the develop, was that the issue?

**Comment by WalterKolczynski-NOAA** [comment]
> @WalterKolczynski-NOAA @EdwardSafford-NOAA the CI seems to be working, but I'm not sure what changed other than merging in the develop, was that the issue?

Maybe, which is kinda why I reran it. Though I don't know why that should've mattered and it still seems like a bug, even if it went away here.

**Comment by JessicaMeixner-NOAA** [comment]
@WalterKolczynski-NOAA anything I need to do on my end?

**Comment by WalterKolczynski-NOAA** [comment]
@JessicaMeixner-NOAA Not now, just waiting for the CI test to complete and go green.

Edit: Oh wait, it did. Shellcheck complaints are trivial and in-line with the surrounding code. Merging.

**Pull request merged**
